mx.pwmc.Svgvi.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "mx.pwmc.Svgvi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "writable-sdk": true,
+    "command": "mx.pwmc.Svgvi",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
```

Comment:
Could you limit it to `xdg-pics`?

Reply:
No. This is a design tool, so it needs user's home directory access.